_posts/2015-04-28-how-I-found-the-cheapest-place-to-live-in-austin.md

```diff
@@ -0,0 +1,274 @@
+not skip a beat at work while being half-stranded in an Airbnb somewhere in
+Austin. 
+
+Sunday arrives, and soon I get a call from Randall.
+
+> Hey Max! It's Randall. I arrived already. Here's the address, come by whenever
+> you can!
+
+Alright! I pack my things, open Google Maps on my phone and put on my earphones 
+and start walking towards what would become home #2. I'm still wondering why I
+didn't take the bus, really, instead of walking for more than an hour to get to
+the place. I don't regret it though, since I had a hefty list of podcasts I
+wanted to go through.
+
+Anyways. I get to the place, confirm that I am indeed at the right complex and
+knock on the door. 
+
+> Max? Hey man, how are you doing? I'm so sorry that I didn't tell you about the
+> bachelor party. It was literally a last minute thing. Come on in!
+
+His apartment was small-ish. A small, really uncomfortable futon (my bed), a 
+table with a DJ set, kitchen, bathroom and the master bedroom. And no WiFi. 
+Crap. Alright, we'll get by. I ended up working out of a coffee shop, and
+cooking eggs in the microwave inside a glass.
+
+It was a really fun experience. He showed me around, drove me places when I
+needed to, and was really eager to give the NomadPad a try. He wanted to 
+establish a network of houses throughout the world that would enable you to
+travel from place to place without having to wonder about leasing fees or other
+rental headaches. We kept talking about it, and in fact one of the reasons he 
+agreed to host me was that he wanted me to be a member of the first house. I was
+totally in. He also shared more of his adventures through Asia and how he became 
+a DJ, later to be chased by the Thai mafia. I'll leave that story for another 
+post.
+
+The original plan was for me to stay during January at Randall's. The NomadPad 
+was scheduled to start in February, and I was ready to move in as soon as we
+figured it all out. However, we soon discovered a flaw in our plans: SXSW.
+
+Now let me backtrack a bit and tell you about SXSW. South by Southwest (SXSW) is
+a set of film, interactive, and music festivals and conferences that take place
+each year in mid-March in Austin. The best way to describe it is this:
+Imagine the whole city being crowded as if it were a concert. That's how many
+people attend the event. Finding a place to stay is incredibly difficult during
+that week, and prices shoot up from $70/night up to $1000/night in some cases.
+It's such a huge deal that some native Austinites leave for a vacation, which is
+paid for by the money they receive from renting their homes.
+
+Now, because Randall still didn't have a house secured for the NomadPad. And he
+needed good prices to make the NomadPad profitable, i.e. find a long term lease.  
+That's why SXSW became a problem.
+
+Instead, Randall decided to change plans and open up the NomadPad in April,
+which left me in a rather awkward situation. I didn't know if I were to be
```

Comment:
I didn't know if I could stay at his place during the next three months or if I had to find a place in the meantime.